tests/store/tracking/test_file_store.py

```diff
@@ -55,6 +58,31 @@ def store(tmp_path):
     return FileStore(str(tmp_path.joinpath("mlruns")))
 
 
+@pytest.fixture
+def store_and_trace_info(store):
+    exp_id = store.create_experiment("test")
+    timestamp_ms = get_current_time_millis()
+    return store, store.start_trace(exp_id, timestamp_ms, {}, {})
+
+
+@pytest.fixture
+def generate_trace_infos(store):
+    exp_id = store.create_experiment("test")
+    timestamps = list(range(0, 100, 10))
+    trace_infos = []
+    request_ids = []
+    for i in range(10):
+        trace_info = store.start_trace(
+            exp_id,
+            timestamps[i],
+            {},
+            {TraceTagKey.TRACE_NAME: f"trace_{i}"},
+        )
+        trace_infos.append(trace_info)
+        request_ids.append(trace_info.request_id)
+    return trace_infos, store, exp_id, request_ids, timestamps
```

Comment:
/ai make the following changes

1. Use `enumerate` to simplify the loop.
2. Define a NamedTuple class and return an instance of it.

Reply:
/ai Nice, can we use `typing.NamedTuple`?